assets/src/containers/GradeDistribution.js

```diff
@@ -31,7 +31,12 @@ const styles = theme => ({
 })
 
 function GradeDistribution (props) {
-  const { classes, disabled, courseId, user } = props
+  const {
```

Comment:
This change seems like inconsistent style with the other components and containers. Is there a reason this style is changed/preferred? I know doesn't like multiple imports on the same line but this project ignores the [E401](https://lintlyci.github.io/Flake8Rules/rules/E401.html) lint suggestion in the backend. ;) Most examples of destructuring seem to keep everything on one line (as long as the line is short) and appears slightly easier to read for me.

Reply:
Good point, I'm reverting it back.